cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java

```diff
@@ -112,7 +112,12 @@ public void clearInternalData(){
     public HashMap getTCData(){
         HashMap tcData = new HashMap();
         for(String key : pref.getAll().keySet()) if (key.startsWith(IABTCF_KEY_PREFIX))
-            tcData.put(key, pref.getString(key, null));
+            try {
```

Comment:
better to make a type check like we do [here](https://github.com/SourcePointUSA/android-cmp-app/blob/76e1c2dc6b402603e8fd7796c360083e5dc514f7/cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java#L43).

Reply:
@leodoin  I have updated this method to remove try catch block